types/operator.go

```diff
@@ -3,24 +3,46 @@ package types
 // OperatorID is a unique ID for the node, used to create shares and verify msgs
 type OperatorID = uint64
 
-// Operator represents an SSV operator node
+// CommitteeMember represents an SSV operator node that is part of a committee
+type CommitteeMember struct {
+	OperatorID        OperatorID
+	CommitteeID       CommitteeID `ssz-size:"32"`
+	SSVOperatorPubKey []byte      `ssz-size:"294"`
+	// FaultyNodes is the number of nodes that are considered faulty or malicious in the operator's committee
+	FaultyNodes uint64
+	// All the members of the committee
+	Committee  []*Operator `ssz-max:"13"`
+	DomainType DomainType  `ssz-size:"4"`
+}
+
+// Operator represents a node in the network that holds an ID and a public key
 type Operator struct {
 	OperatorID        OperatorID
-	SharePubKey       []byte `ssz-size:"48"`
-	SSVOperatorPubKey []byte `ssz-size:"294"`
+	SSVOperatorPubKey []byte `ssz-size:"459"`
 }
```

Comment:
`CommitteeMember.SSVOperatorPubKey` and `Operator.SSVOperatorPubKey` should have the same ssz-size. Since we changed it from 294 to 459, we need to also update `CommitteeMember.SSVOperatorPubKey`. Actually, it's even possible to insert an attribute of type `Operator` in `CommitteeMember`, if wanted.

Reply:
fixing SSZ size...
The other option is good... but I prefer not to refactor now before merge